iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergInputFormat.java

```diff
@@ -178,7 +178,7 @@ public RecordReader<Void, Container<Record>> getRecordReader(InputSplit split, J
 
   @Override
   public boolean shouldSkipCombine(Path path, Configuration conf) {
-    return true;
+    return false;
```

Comment:
to disable combine functionality we could use hive.merge.tezfiles=false, right?

Reply:
This function is used during generation of splits in CombineRecordReader (during generation of merge tasks). It is a flag function which tells whether merging is supported.
`hive.merge.tezfiles=false` will still work to disable merge functionality.

Reply:
👍  i was just asking if we have flag to disable merge functionality.